diff --git a/crates/extensions/tedge_flows/Cargo.toml b/crates/extensions/tedge_flows/Cargo.toml
index 51d7ba14b..f28173104 100644
--- a/crates/extensions/tedge_flows/Cargo.toml
+++ b/crates/extensions/tedge_flows/Cargo.toml
@@ -14,7 +14,7 @@ async-trait = { workspace = true }
 camino = { workspace = true, features = ["serde1"] }
 futures = { workspace = true }
 humantime = { workspace = true }
-rquickjs = { git = "https://github.com/thin-edge/rquickjs", rev = "4ed04ac9af3de453dd41ff09fdd1837c7ceb1f1c", default-features = false, features = [
+rquickjs = { git = "https://github.com/rina23q/rquickjs", rev = "d5db616760878c9d27f1d9b7fc37b8d572d97948", default-features = false, features = [
     # disable bindgen and rely on pre-generated bindings due to problems
     # generating the bindings at build time with clang
     #"bindgen",
